### ISSUE TITLE
Failed WCAG contrast? Suggest values to fix

### ISSUE THREAD

**Issue by lloydi** [superhighfives/pika#65]
Alrighty, another thing to mull over :)

If the two colours fail WCAG contrast levels, provide a fix/adjust icon which, when activated, provides options to fix the contrast be either darkening the darker of the two colours until it passes, or lightening the lighter of the two until contrast ratio is good. Where one of the colours cannot be made any darker or lighter, then it'd have to be an adjustment to both.

What do you think?

**Comment by superhighfives** [comment]
I think this ties into a suite of enhancements that I'd love to make, but that I'm unsure I have the bandwidth to implement and maintain. Currently Pika is very un-opinionated, and only represents the colours that the user picks—for example, you can't enter your own hex code (#57), or keep a history (#26). There are requests for those kinds of features, which would make it a more fully fledged colour management tool, but I'm not sure whether or not that makes sense in terms of the vision of the app as a tool to quickly and easily check colours onscreen, and it would be considerable work to implement in terms of the code and the additional UI complexity.

That said, I'd love to keep this open as a potential one for the roadmap, and see if it gets more thumbs up—especially as the amount of time I can commit may change in future.

Thanks for the suggestion!

**Comment by superhighfives** [comment]
You can now tweak colours until you get a result that passes. It's currently in beta, but you can grab it via the app or at https://github.com/superhighfives/pika/releases/tag/0.0.18-beta2. See more on #125.

**Comment by lloydi** [comment]
Perfect! Just what I had in mind, love it!
